example/android/app/build.gradle

```diff
@@ -100,6 +100,6 @@ android {
 
 dependencies {
     compile "com.android.support:appcompat-v7:23.4.0"
-    compile "com.facebook.react:react-native:0.29.0"  // From node_modules
+    compile "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
I don't understand why people continuously want to do this... Using `+` is **not** okay